OrbitGl/TimeGraph.cpp

```diff
@@ -317,8 +317,12 @@ void TimeGraph::ProcessTimer(const Timer& a_Timer) {
     track->OnTimer(a_Timer);
     ++m_ThreadCountMap[a_Timer.m_TID];
   } else {
-    // Use thead 0 as container for scheduling events.
-    GetThreadTrack(0)->OnTimer(a_Timer);
+    // Use thread 0 as container for scheduling events.
+    const std::shared_ptr<ThreadTrack>& track0 = GetThreadTrack(0);
+    track0->SetName("(Select all threads)");
```

Comment:
Suggestions are welcome.

Reply:
Just "All threads"?

Reply:
We could add the process name.  Will be useful when we'll support multiple processes.

Reply:
Sounds good to me!

Reply:
So now it looks like `<process_name> (all threads)`.